keras/utils/data_utils.py

```diff
@@ -110,6 +124,15 @@ def dl_progress(count, block_size, total_size):
 
 
 def validate_file(fpath, md5_hash):
```

Comment:
This is a single-usage util function and should be left out (e.g. added to `EXCLUDE` in `autogen.py`.